paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -14,18 +14,28 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
     var thisClass = this;
 
     var popUp = jQuery('<div id="dce-info-popup"></div>');
-    var buttonActions =[ 'Help with this player', 'Report a problem', 'Feedback', 'All Course Videos' ];
+    var buttonActions =[ 'Help with this player', 'Report a problem', 'DCE system status', 'Feedback', 'All Course Videos' ];
 
+    jQuery(popUp).append(thisClass.getItemTitle());
```

Comment:
It's not a question of syntax. It's redundant code. `var popUp = JQuery(...)` initializes `popUp` as a jQuery object on line 16. Wrapping it in `jQuery()` again on line 18 is pointless.